conf/app.routes

```diff
@@ -149,3 +149,18 @@ POST        /which-previous-registration-vat-period-to-pay
 
 GET        /vat-payable-check/:periodIndex/:countryIndex                              controllers.corrections.CheckVatPayableAmountController.onPageLoad(waypoints: Waypoints ?= EmptyWaypoints, periodIndex: Index, countryIndex: Index)
 POST       /vat-payable-check/:periodIndex/:countryIndex                              controllers.corrections.CheckVatPayableAmountController.onSubmit(waypoints: Waypoints ?= EmptyWaypoints, periodIndex: Index, countryIndex: Index, incompletePromptShown: Boolean)
+
+GET         /identity-problem                                             controllers.auth.IdentityVerificationController.identityError(redirectUrl: RedirectUrl)
+GET         /identity-complete                                            controllers.auth.IdentityVerificationController.handleIvFailure(redirectUrl: RedirectUrl, journeyId: Option[String])
+GET         /iv/error                                                     controllers.auth.IvReturnController.error(continueUrl: RedirectUrl)
+GET         /iv/incomplete                                                controllers.auth.IvReturnController.incomplete(continueUrl: RedirectUrl)
+GET         /iv/insufficient-evidence                                     controllers.auth.IvReturnController.insufficientEvidence(continueUrl: RedirectUrl)
+GET         /iv/locked-out                                                controllers.auth.IvReturnController.lockedOut(continueUrl: RedirectUrl)
+GET         /iv/precondition-failed                                       controllers.auth.IvReturnController.preconditionFailed(continueUrl: RedirectUrl)
+GET         /iv/technical-issue                                           controllers.auth.IvReturnController.technicalIssue(continueUrl: RedirectUrl)
+GET         /iv/timeout                                                   controllers.auth.IvReturnController.timeout(continueUrl: RedirectUrl)
+GET         /iv/user-aborted                                              controllers.auth.IvReturnController.userAborted(continueUrl: RedirectUrl)
+GET         /iv/insufficient-evidence-sources                             controllers.auth.IvReturnController.notEnoughEvidenceSources(continueUrl: RedirectUrl)
+GET         /iv/failed-matching                                           controllers.auth.IvReturnController.failedMatching(continueUrl: RedirectUrl)
+GET         /iv/failed-identity-verification                              controllers.auth.IvReturnController.failed(continueUrl: RedirectUrl)
```

Comment:
We can align vertically the new routes with the existing ones

Reply:
I tried to reflect what you suggested